pkg/config/templates/rancherd-12-monitoring-dashboard.yaml

```diff
@@ -1136,15 +1136,15 @@ resources:
             "steppedLine": false,
             "targets": [
               {
-                "expr": "irate(kubevirt_vmi_storage_read_times_ms_total{namespace=\"$namespace\", name=\"$vm\"}[5m])",
+                "expr": "irate(kubevirt_vmi_storage_read_times_seconds_total{namespace=\"$namespace\", name=\"$vm\"}[5m])",
                 "format": "time_series",
                 "interval": "",
                 "intervalFactor": 1,
                 "legendFormat": "{{drive}}-read",
                 "refId": "A"
               },
               {
-                "expr": "irate(kubevirt_vmi_storage_write_times_ms_total{namespace=\"$namespace\", name=\"$vm\"}[5m])",
+                "expr": "irate(kubevirt_vmi_storage_write_times_seconds_total{namespace=\"$namespace\", name=\"$vm\"}[5m])",
```

Comment:
The unit changes from `ms` to `s`. Do we need to modify the format in the chart?
<img width="633" alt="截圖 2024-04-03 晚上7 07 37" src="https://github.com/harvester/harvester-installer/assets/1691518/eb52f8ef-11e8-417b-9f57-fbcf0d901337">

Reply:
cc @brandboat to check too

Reply:
With the new commit, it is as below now.

![image](https://github.com/harvester/harvester-installer/assets/31133476/b46a745a-cd1c-46ad-9c71-7175c1243175)

Reply:
Yes, the format should be changed to `s` since now `kubevirt_vmi_storage_read_times_seconds_total` and `kubevirt_vmi_storage_write_times_seconds_total` both use second as the unit instead of ms.

Related libvirt code is here, https://github.com/kubevirt/kubevirt/blob/44c8c970faa9d514d82e479e17f0edce2eb21731/pkg/monitoring/domainstats/prometheus/prometheus.go#L368-L388, both `block.RdTimes` and `block.WrTimes` are in `ns` unit and converted into `s` unit, this change was introduced by commit  https://github.com/kubevirt/kubevirt/commit/dfdd479649b7cf630aa43981730c07fab92c0239.

More info about `block.RdTimes` and `block.WrTimes` , please check
https://libvirt.org/manpages/virsh.html